packages/lit-dev-content/samples/tutorials/intro-to-lit/01/before/my-element.js

```diff
@@ -1 +1,2 @@
-export {};
+import {LitElement, html} from 'lit';
+import {customElement} from 'lit/decorators.js';
```

Comment:
```suggestion
import {LitElement, html} from 'lit';
```

Oops. Might've been in a bit of a hurry there.